lab3/lab3_src.c

```diff
@@ -17,7 +17,7 @@ int main(void)
     FILE * load_file;
     char line[256];
     float temp_threshold = LOAD_AVG_MAX_RESOLUTION;
-    float pin_thresholds[8];
+    float pin_thresholds[7];
```

Comment:
@alyssawardwell The number in the brackets should be the number of elements in the array.  In our case we want one threshold per pin, which would be 8.

When we access each member of the array the index goes from 0-7 for 8 total members